### PR TITLE
Relay machine name in lamassu-update

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Relay machine name in lamassu-update](https://trello.com/c/x7ZL68cm/608-relay-machine-name-in-lamassu-update)

File: lib/update/updater.js [hunk 1]
```diff
@@ -181,9 +181,23 @@ Updater.prototype._update = function _update () {
...
 
+Updater.prototype.updateHeaders = function updateHeaders (options) {
+  var dataPath = path.resolve(__dirname, '..', this.config.dataPath)
+  var machineInfo = this.machineInfo.load(dataPath)
+
+  if (machineInfo.active) {
+    return _.assign(options, {
+      'device-machine-id': machineInfo.deviceId,
+      'device-machine-name': machineInfo.deviceName
+    })
```
Comment (csrapr) [inline lib/update/updater.js:192]: what's the reason behind having -machine- in the mddle? We usually call machines "devices" throughout our codebase

**Review by csrapr — CHANGES_REQUESTED** [review]

File: lib/update/updater.js (hunk 1 above)
Comment (josepfo) [reply lib/update/updater.js:192]: httpsOptions.headers already have a property 'device-id', the -machine- is pretty much to differentiate

**Review by RafaelTaranto — APPROVED** [review]

File: lib/trader.js [hunk 2]
```diff
@@ -407,6 +409,8 @@ Trader.prototype.pollHandler = function pollHandler (res) {
     this.triggers = createCompatTriggers(res)
   }
 
+  machineInfo.save(this.dataPath, res.machineInfo)
```
Comment (joshmh) [inline lib/trader.js:412]: Shouldn't we be using ``await`` here?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/trader.js (hunk 2 above)
Comment (RafaelTaranto) [reply lib/trader.js:412]: Machine info isn't required for the machine to work, so we can move past this without waiting. Also this is inside the polling function, waiting for FS work should be prevented.

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**